src/ipc.h

```diff
@@ -33,6 +33,7 @@ namespace FdoIPC {
 class MessageReceiver {
 public:
     virtual void didReceiveMessage(uint32_t messageId, uint32_t messageBody) = 0;
+    virtual ~MessageReceiver() = default;
```

Comment:
Nit: the defaulted virtual constructor fits better at the beginning of this struct declaration.

Reply:
Changed!